Jenkinsfile

```
@@ -107,7 +107,9 @@ pipeline {
             } 
             steps {
                 script {
-                    dockerStack.deploy(env.STACK_NAME, 'docker-stack.yml')
+                    timeout(time: 10, unit: 'MINUTES') {
```

I can't seem to easily see stage timings going back that far, but at least one successful job in the recent past took 15 minutes total, so I think it's plausible that the deployment stage could sometimes run longer than 10. How about we go with 15 here?

```suggestion
                    timeout(time: 15, unit: 'MINUTES') {
```